velox/functions/prestosql/tests/ArraySortTest.cpp

```diff
@@ -405,6 +405,39 @@ TEST_P(ArraySortTest, constant) {
   assertEqualVectors(expected, result);
 }
 
+TEST_P(ArraySortTest, multilevelDictionaries) {
+  if (GetParam() != TypeKind::BIGINT) {
+    GTEST_SKIP() << "Skipping constant test for non-bigint type";
```

Comment:
@xiaoxmeng fbcode doesn't like GTEST_SKIP for some reason. Would be nice to refactor these tests to not use GTEST_SKIP.